eth-contracts/contracts/Staking.sol

```diff
@@ -354,6 +375,19 @@ contract Staking is InitializableV2 {
         return totalStakedHistory.getLast();
     }
 
+    /**
+     * @notice Get the total amount of tokens staked by `_accountAddress` at block number `_blockNumber`
+     * @param _accountAddress - Account requesting for
+     * @param _blockNumber - Block number at which we are requesting
+     * @return The amount of tokens staked by the account at the given block number
+     */
+    function totalStakedForAt(
+        address _accountAddress,
+        uint256 _blockNumber
+    ) public view returns (uint256) {
```

Comment:
this needs to be changed from external to public because it's used inside the contract